README.md

```diff
@@ -6,22 +6,45 @@
 
 ## Installation
 
+### Prerequisities
+
+* GNU-Make
+* Python && pip
+* OpenJDK (required by submodule)
+* optional: Docker CE for building container images 
+
 Clone the repository:
 
 ```sh
-git clone ocrd_fileformat
-cd ocrd_fileformat
+git clone --recursive git@github.com:OCR-D/ocrd_fileformat.git
 ```
 
-Update submodules:
```

Comment:
I would still keep that information because what if user didn't use the recursive flag when cloning.

Reply:
I added verbosity so every willing reader should realize why the flag is included. After all, this saves you from typing two more lines ;)

Reply:
Willing perhaps, but able? :-P